src/components/pages/profile/[name]/registration/steps/Pricing/TemporaryPremium.tsx

```diff
@@ -0,0 +1,580 @@
+import { CalendarEvent, google, ics, office365, outlook, yahoo } from 'calendar-link'
+import {
+  ChangeEventHandler,
+  RefObject,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { DefaultTheme, css } from 'styled-components'
+
+import { Button, Dropdown, Helper, Input, Typography, mq } from '@ensdomains/thorin'
+
+import CalendarSVG from '@app/assets/Calendar.svg'
+import MobileFullWidth from '@app/components/@atoms/MobileFullWidth'
+import { makeDisplay } from '@app/utils/currency'
+
+const VAR_PREFIX = '--premium-chart-'
+const startPrice = 100000050.0
```

Comment:
startPrice is kinda misleading here because its actually just a variable in the exponential func, real startPrice is 100M - 50

Reply:
oh okay technically it starts at 100mil minus $47, although i'm not sure it makes too much difference since the price changes so quick. It would make a bigger difference closer to the end date

Reply:
so its -47 instead of -50?